docs/content/preview/architecture/transactions/read-committed.md

```diff
@@ -27,24 +27,24 @@ The following two key semantics set apart Read Committed isolation from Repeatab
 
-  Note that the clock skew between all nodes in the cluster is always in a [`max_clock_skew`](/preview/reference/configuration/yb-tserver/#max-clock-skew-usec) bound due to clock synchronization algorithms.
-* For writes at some time later than `T1` + `max_clock_skew`, the database can be sure that they were done after the read timestamp was chosen on any node. But for writes at a time between `T1` and `T1` + `max_clock_skew`, node `N2` can find itself in an ambiguous situation such as the following:
+  Note that the clock skew between all nodes in the cluster is always in a [max_clock_skew_usec](/preview/reference/configuration/yb-tserver/#max-clock-skew-usec) bound due to clock synchronization algorithms.
+* For writes with a write timestamp higher than `T1` + `max_clock_skew`, the database can be sure that they were done after the read timestamp was chosen. But for writes at a time between `T1` and `T1` + `max_clock_skew`, node `N2` can find itself in an ambiguous situation such as the following:
 
-  * It should still return the data if the client issued the read after the data was committed, because it could be the same client connecting to YSQL from a different node and the following guarantee needs to be maintained; the database always returns data that was committed in the past.
+  * It should return the data even if the client issued the read from a different node after writing the data since the following guarantee needs to be maintained: the database always returns data that was committed in the past (here past means the user-perceived past not based machine clocks).
 
   * It should not return the data if the write was actually performed in the future, that is, after the read point (also known as snapshot) was chosen.
 
-* If node `N2` finds writes in the range `(T1, T1+max_clock_skew]`, to avoid breaking the strong guarantee that a reader should always be able to read what was committed earlier, node `N2` avoids giving incorrect results and raises a `Read restart` error.
+* Hence, if node `N2` finds writes in the range `(T1, T1+max_clock_skew]`, to avoid breaking the strong guarantee that a reader should always be able to read what was committed earlier, and to also avoid reading data that was actually written after the read was issued, node `N2` raises a `Read restart` error.
 
 Some distributed databases handle this uncertainty due to clock skew by using algorithms to maintain a tight bound on the clock skew, and then taking the conservative approach of waiting out the clock skew before acknowledging the commit request from the client for each transaction that writes data. YugabyteDB instead uses various mechanisms internally to reduce the scope of this ambiguity, and if there is still ambiguity in rare scenarios, the error is surfaced to the client.
```

Comment:
Suggestion:

Some distributed databases handle uncertainty due to clock skew by using algorithms to maintain a tight bound on the clock skew, and then taking the conservative approach of waiting out the clock skew before acknowledging the commit request from the client for each transaction that writes data. YugabyteDB, however, uses various internal mechanisms to reduce the scope of this ambiguity, and if there is still ambiguity in rare cases, the error is surfaced to the client.

Reply:
changed `this certainty` to `this specific uncertainty` instead of removing this because that is important. There could be many other uncertainties but we want to refer this one. Let me know what you think